river/Config.zig

```diff
@@ -73,8 +76,14 @@ pub fn init(self: *Self) !void {
     self.float_filter = std.ArrayList([*:0]const u8).init(util.gpa);
     errdefer self.float_filter.deinit();
 
+    self.csd_filter = std.ArrayList([*:0]const u8).init(util.gpa);
```

Comment:
This should just store a slice. (the float filter should as well but that'd be out of scope for this)
```suggestion
    self.csd_filter = std.ArrayList([]const u8).init(util.gpa);
```